qa/c8-orchestration-cluster-e2e-test-suite/utils/requestHelpers/cluster-variable-requestHelpers.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {expect, APIRequestContext} from '@playwright/test';
+import {
+  buildUrl,
+  jsonHeaders,
+  assertStatusCode,
+  extractAndStoreIds,
+} from '../http';
+import {defaultAssertionOptions} from '../constants';
+import {CREATE_CLUSTER_VARIABLE} from '../beans/requestBeans';
+
+/**
+ * Creates a global cluster variable and stores the response fields in state
+ */
+export async function createGlobalClusterVariable(
+  request: APIRequestContext,
+  state: Record<string, unknown>,
+  stateKey: string,
+  variableData?: {name: string; value: unknown},
+): Promise<void> {
+  const data = variableData || CREATE_CLUSTER_VARIABLE();
+
+  await expect(async () => {
+    const res = await request.post(buildUrl('/cluster-variables/global'), {
+      headers: jsonHeaders(),
+      data,
+    });
+    await assertStatusCode(res, 200);
+    await extractAndStoreIds(res, state);
+    const json = await res.json();
+    state[`${stateKey}Name`] = json.name;
+    state[`${stateKey}Value`] = json.value;
+    state[`${stateKey}Scope`] = json.scope;
+  }).toPass(defaultAssertionOptions);
```

Comment:
I think this endpoint is strongly consistent therefore `expect().toPass()` shouldn't be necessary. If I am wrong, thats fine